sugarscape.py

```diff
@@ -34,11 +34,13 @@ def __init__(self, configuration):
                                     "universalSugarIncomeInterval": configuration["environmentUniversalSugarIncomeInterval"],
                                     "sugarscapeSeed": configuration["seed"]}
         self.seed = configuration["seed"]
```

Comment:
Moved initialization of `self.debug` above initialization of `environment` so that the `Environment` can have its own `self.debug`.